packages/bff/src/model/domain/errors.ts

```diff
@@ -6,12 +6,33 @@ import {
 
 export const errorCodes = {
   purposeNotFound: "0001",
+  userNotFound: "0001",
+  selfcareEntityNotFilled: "0032",
 };
 
 export type ErrorCodes = keyof typeof errorCodes;
 
 export const makeApiProblem = makeApiProblemBuilder(errorCodes);
 
+export function selfcareEntityNotFilled(entity: string): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Selfcare entity not filled for ${entity}`,
```

Comment:
The error here is slightly different `Selfcare entity $className with field $field not filled`